src/style.scss

```diff
@@ -363,21 +366,22 @@ input {
 
 :host([bordered]) {
   .inputElement {
-    padding-right: var(--auro-size-150, .75rem);
-    padding-left: var(--auro-size-150, .75rem);
-    border: 1px solid var(--auro-color-border-primary-on-light);
-    border-radius: var(--auro-border-radius);
-    background-color: var(--auro-color-base-white);
+    padding-right: var(--ds-size-150, .75rem);
+    padding-left: var(--ds-size-150, .75rem);
+    border: 1px solid var(--ds-color-border-primary-default, $ds-color-border-primary-default);
+    border-radius: var(--ds-border-radius, $ds-border-radius);
+    background-color: var(--ds-color-base-white, $ds-color-base-white);
 
     &:focus {
-      border-color: var(--auro-color-ui-default-on-light);
+      border-color: var(--ds-color-ui-default-default, $ds-color-ui-default-default);
 
       @include auro_breakpoint--sm {
-        box-shadow: inset 0 0 0 1px var(--auro-color-ui-default-on-light);
+        box-shadow: inset 0 0 0 1px var(--ds-color-ui-default-default, $ds-color-ui-default-default);
       }
     }
 
     &:disabled {
+      // where does this "auro-color-alert-disabled-on-light" refers to?
       border-color: var(--auro-color-alert-disabled-on-light);
```

Comment:
Hi @jason-capsule42, need advise on this token value.

Reply:
There is no disabled version of this. Looks like there was a bug and it should have been `--auro-color-alert-error-default`. Replace it basded on that.